schema/generate/60-schema_update_62.sql

```diff
@@ -1,42 +0,0 @@
-BEGIN; 
-
```

Comment:
Hey @sgoggins, Sorry I am not updated with the recent changes in Augur, but would deleting the schema affect the libyear and scorecard?

Reply:
@Dhruv-Sachdev1313 : I don't think I am deleting the schema. :)